.gitignore

```diff
@@ -23,14 +23,19 @@
 # Ignore Byebug command history file.
 .byebug_history
 
+# Ignore the coverage folder
+SIGS/coverage/*
+SIGS/coverage/
+
 # Other files
 .vagrant
 .gitignore
-SIGS/coverage
 SIGS/script/dbScript
 SIGS/.generators
 SIGS/.rakeTasks
 SIGS/db/schema.rb
 
 # Ignore IDE files
 .idea/
+
```

Comment:
Indique qual tipo do arquivo ignorado ou coloque o .DS_Store no _Other files_

Reply:
Just add a new commit solving  this  problem, thanks for this review! :)